### PR TITLE
Add copyright comments

### PR DESCRIPTION
**Pull request by hobyst**
This PR should improve Vita3K's license compliance with both its own GPL v2
license ~~and the licenses every other 3rd-party software Vita3K includes and/or
uses~~ by solving issue #1282, ~~adding a 3rd-party license notice and making this
last file along with the license of the project itself to be copied and included
in every further compilation of the software~~.

### Adding copyright notices to source files
The year assigned on the new copyright notices has been assigned based on the
year of the most recent commit. Some of them have 2019 as their year due to
a project reorganization done by Zangetsu in 2019 which removed all traces
from possible past commits in the version control tree.

~~Copies of the MIT license and the corresponding ImGui copyright notice have
been added to the in-repo copies of the backend implementations copied from
the original ImGui repository as well.~~

### 3rd-party license notice

**DROPPED FOR ANOTHER PR**

- Discord Game SDK: Couldn't find license
- FFmpeg: Among the possible options, the adopted license is unknown.
  Options are:
   - GPL v2
   - GPL v3
   - LGPL v2.1
   - LGPL v3
- Google Test: Is Google Test included in the binary distribution of Vita3K?
  Testing tools aren't supposed to be included in binary distributions, but
  it's better to confirm it just in case.
- imgui_club: Are any of the tools included there actually
  incorporated into Vita3K?
- libtomcrypt: Among the possible options, the adopted license is unknown.
  Options are:
   - Public domain
   - DWTFYWTPL
- Microprofile: Is microprofile incorporated into Vita3K or is it
  only used for development?
- psvpfstools: No license notice is provided on the [repository](https://github.com/motoharu-gosuto/psvpfstools)
  for the actual source code of the library.
- rpscs3: Its license requires a copyright notice, but none was found.
- stb: Among the possible options, the adopted license is unknown.
  Options are:
   - MIT license
   - The Unlicense
- Unicorn: Its license requires a copyright notice, but none was found.

### CODE REVIEW

**Comment by sunho** [comment]
image

lmao

**Comment by sunho** [comment]
Thanks for putting time into this.

Google Test is only used for development purpose. 
Microprofile is embeded in the binary of Vita3k. 
Discord Game SDK and psvpfstools have zero information on its licenses.
For rpcs3, I'm not really sure. You can talk with the rpcs3 team directly if we need to. 

For other libraries like ffmpeg, stb, or imgui, they are widely used in other open sources projects. You might look at how those projects are doing.

I don't personally feel the need to clarify license stuffs this much. I'm not a lawyer, so I can't speak much. It's better to be safe after all.

If you find it difficult to sort out license stuffs, you can this PR and handle licenses later. We can merge copywrite comments part right away.

File: vita3k/Vita3K.rc [hunk 1]
```diff
@@ -1,3 +1,20 @@
+// Vita3K emulator project
+// Copyright (C) 2019 Vita3K team
```
Comment (Zangetsu38) [inline vita3k/Vita3K.rc:2]:
```suggestion
// Copyright (C) 2021 Vita3K team
```
all time keep year in current year :)

**Comment by hobyst** [comment]
I have dropped the 3rd-party license commit and stashed it on my side and run clang-format so now checks should be fine.

File: vita3k/Vita3K.rc (hunk 1 above)
Comment (hobyst) [reply vita3k/Vita3K.rc:2]: Should I update the copyright year to 2021 only on the source code files I placed a copyright notice on or on every source code file on the repository?

**Comment by hobyst** [comment]
> I don't personally feel the need to clarify license stuffs this much. I'm not a lawyer, so I can't speak much. It's better to be safe after all.

The thing is that closely reading all the licenses is clear that most of them require both a copyright notice and a license copy on any code distribution in both source and binary format. [Discord](https://discord.com/licenses), [Apple](https://www.apple.com/legal/internet-services/maps/oss-acknowledgements/), [Nintendo](https://www.nintendo.co.jp/support/oss/) and even Sony for the [PS Vita](https://doc.dl.playstation.net/doc/psvita-oss/index.html), [PS4](https://doc.dl.playstation.net/doc/ps4-oss/index.html) and [PS5](https://doc.dl.playstation.net/doc/ps5-oss/index.html) just to name a few have to make this kind of document and include them somewhere in their apps or devices, usually on the settings under some sort of "Credits", "Acknowledgements" or "OSS Credits".

File: vita3k/Vita3K.rc (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/Vita3K.rc:2]: hmm i thinks using like some other i have see update copyright on this year for all file project

**Comment by hobyst** [comment]
This should be it. As this gets merged, I'll create a follow up PR for the 3rd-party license aknowledgements.

**Comment by IllusionMan1212** [comment]
Would something like `Copyright (C) 2018-2021 Vita3K team` make more sense?

**Review by IllusionMan1212 — APPROVED** [review]
LGTM. i'll leave the merging to you or other devs

**Pull request merged**